provisioning/roles/ruby/tasks/main.yml

```
@@ -6,7 +6,10 @@
     version: v1.1.0
 
 - name: try to compile dynamic bash extension to speed up rbenv
-  shell: cd ~/.rbenv && src/configure && make -C src
+  command: src/configure && make -C src
+  ignore_errors: yes
```

Why do you need to ignore errors?